app/assets/javascripts/views/story_view.js

```diff
@@ -254,13 +256,33 @@ module.exports = FormView.extend({
     this.$el.appendTo(this.model.get('column'));
   },
 
+  setStoryDetailRequest: function() {
+    const shouldRequest = !this.model.attributes.isFetched;
+    if (shouldRequest) {
+      this.fetchStoryDetails();
+    }
+  },
+
   startEdit: function(e) {
     if (this.eventShouldExpandStory(e)) {
       this.model.set({editing: true, editingDescription: false, clickFromSearchResult: this.$el.hasClass('searchResult')});
+      this.setStoryDetailRequest();
       this.removeHoverbox();
+      this.render();
     }
   },
 
+  fetchStoryDetails: function() {
+    var that = this;
+    return axios
+      .get(`/projects/${this.model.collection.project.id}/stories/${this.model.id}`)
+      .then(({ data }) => changeCase.camelKeys(data, {recursive: true, arrayRecursive: true}))
+      .then(({ story }) => {
+        that.model.set(story)
+        that.model.set({isFetched: true})
```

Comment:
Same thing here about the arrow functions AND you missed some spaces 💅 

```suggestion
        this.model.set({ isFetched: true })
```